node/full_client.go

```diff
@@ -60,6 +60,10 @@ func NewFullClient(node *FullNode) *FullClient {
 	}
 }
 
+// GetClient returns a new RPC client for the full node.
+//
+// TODO: should this be NewRPPCClient? Or should we add the client as a field of
+// the FullNode so that it is just created once?
```

Comment:
Thoughts on this? imo we should add the client as a field to the node if possible so that we aren't re-allocating if we call this method more than once. Also then it is a more accurate name.